README.md

```diff
@@ -21,9 +21,27 @@ library and the example project.
 
 ### Steps
 1. In the directory _java_ run the command `mvn package`.
-2. After completion the directory _java/target_ wil contains the example executable and libraries.
+2. After completion the directories _java/client/target_ and _java/examples/target_ will contains the example executable and libraries.
 3. For running the example execute `java -jar examples-2.0-SNAPSHOT.jar -token <access-token> -url <api-url>`.
 
+### Usage generated source 
+One approach is to include the generated sources, which after running the previous build steps, can be found in the directory 
+_java/client/target/generated-sources/openapi/src/main/java_.
+Include or copy this directory into your project and mark it in the IDE as Java source.
```

Comment:
I did something else: copy the api-client Maven project to my own project. Can you describe that as well?